docs/developer/releasing.md

```diff
@@ -16,18 +16,24 @@
 
 ## Process
 
-1. Create a pull request that contains two changes:
+For version 0.x.y:
+
+1. We will target a branch called `release-0.x`.  If this is `0.x.0` then we'll
+   create a branch from master using `git push origin master:release-0.x`, otherwise
+   simply checkout the existing branch `git checkout release-0.x`
+2. Make two changes:
    1. Change [the cluster api controller manager image
    tag][managerimg] from `:latest` to whatever version is being released
-   2. Change the `CONTROLLER_IMAGE` variable in the [Makefile][makefile] to the
+   2. Change the `CONTROLLER_IMG` variable in the [Makefile][makefile] to the
       version being released
-2. Get the pull request merged
-3. From the commit in step 1 (that is now in the master branch), build and push
-   the container images and fat manifest with `make all-push`
-4. Create a tag from this same commit and push the tag to the github repository
-5. Revert the commit made in step 1
-6. Open a pull request with the revert change
-7. Get that pull request merged
+   (Note that we do not release the example-provider image, so we don't tag that)
+3. Commit it using `git commit -m "Release 0.x.y"`
+4. Submit a PR to the `release-0.x` branch, e.g. `git push $USER; hub pull-request -b release-0.x`
+5. Get the pull request merged
+6. Switch to the release branch and update to pick up the commit.  (e.g. `git
+   checkout release 0.x && git pull`).  From there build and push the container
+   images and fat manifest with `make all-push`
```

Comment:
Looks good for the general case, but I didn't backport the multi-arch images to the `release-0.1` branch, so you might have to use the old make target(s).